assets/img/svg/release-level-banner--unstable.svg

```diff
@@ -1,3 +1,12 @@
+<!--*- coding: utf-8 -*- svg -*- ../svg/release-level-banner--unstable.svg *****
```

Comment:
unsure whether we were aware this would become an issue further down the line&hellip;

> [&hellip;] lead to double hyphens in SVG comments which is not valid.
>
> &mdash;https://qcad.org/bugtracker/index.php?do=details&task_id=2101